CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Changelog
   * bfd (@saichint)
 * OSPF
   * ospf_area (@saichint)
+  * ospf_area_vl (@saichint)
```

Comment:
I know the name is getting long. I'd prefer more than just an abbreviation though. How does `ospf_area_vlink` sound?